listenstore/bin/listenstore-kafka-to-cassandra-writer.py

```diff
@@ -13,6 +15,12 @@ def __init__(self):
     def run(self):
         self.kafkaConsumer.start_listens(self.listenStore)
 
+    @property
+    def listenStore(self):
```

Comment:
Well, you can put that in a separate commit if you don't want to mix things together. But that's the convention we use in all of our projects.